src/modules/sdlog2/sdlog2_messages.h

```diff
@@ -185,8 +185,20 @@ struct log_STAT_s {
 	float load;
 };
 
+/* --- CONTROL STATE --- */
+#define LOG_CTS_MSG 11
```

Comment:
I skipped 46 because I was unsure about the comment at line 480: 
```
/* WARNING: ID 46 is already in use for ATTC1 */
```
When adding to the end of the list, should I add it as 47 or 46 and modify the comment?